### PR TITLE
Refactor Person to Patient

### LINKED ISSUE #32: Refactor 'Person' to 'Patient'

### PR DESCRIPTION
**Pull request by divakarmal**
Closes #32

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any patient obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
 in the Software without restriction, including without limitation the rights
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
+copies of the Software, and to permit patients to whom the Software is
```
Comment (foojingyi) [inline LICENSE]: Should this document need to be refactored?

File: docs/tutorials/AddRemark.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ package seedu.address.logic.commands;
 import seedu.address.model.Model;
 
 /**
- * Changes the remark of an existing person in the address book.
+ * Changes the remark of an existing patient in the address book.
```
Comment (foojingyi) [inline docs/tutorials/AddRemark.md]: Should we edit the tutorial document?

File: docs/tutorials/RemovingFields.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Fortunately, IntelliJ IDEA provides a robust refactoring tool that can identify
 
 ### Assisted refactoring
```
Comment (foojingyi) [inline docs/tutorials/RemovingFields.md]: Perhaps this document does not have to be edited as well?

File: docs/tutorials/TracingCode.md [hunk 4]
```diff
@@ -152,14 +152,14 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
    @Override
    public CommandResult execute(Model model) throws CommandException {
        ...
-       Person personToEdit = lastShownList.get(index.getZeroBased());
-       Person editedPerson = createEditedPerson(personToEdit, editPersonDescriptor);
-       if (!personToEdit.isSamePerson(editedPerson) && model.hasPerson(editedPerson)) {
+       Person patientToEdit = lastShownList.get(index.getZeroBased());
+       Person editedPatient = createEditedPerson(patientToEdit, editPatientDescriptor);
+       if (!patientToEdit.isSamePerson(editedPatient) && model.hasPerson(editedPatient)) {
```
Comment (foojingyi) [inline docs/tutorials/TracingCode.md]: Is this part of a tutorial as well? Should it be changed?

File: src/main/java/seedu/address/model/AddressBook.java [hunk 5]
```diff
@@ -53,68 +53,69 @@ public void setPersons(List<Person> persons) {
...
+        patients.remove(key);
     }
 
     //// util methods
 
     @Override
     public String toString() {
-        return persons.asUnmodifiableObservableList().size() + " persons";
+        return patients.asUnmodifiableObservableList().size() + " patients";
         // TODO: refine later
```
Comment (foojingyi) [inline src/main/java/seedu/address/model/AddressBook.java:102]: Perhaps we should keep this in view?

**Review by foojingyi — COMMENTED** [review]
Good refactoring, should not have any compilation or grammatical errors. Just need to take note of which files are being changed.

**Review by PrestonTYR — APPROVED** [review]
lgtm

**Pull request merged**